src/event_statistics.cpp

```diff
@@ -203,6 +206,13 @@ struct value_constraint {
                       name, equals_statistic_->str() );
         }
     }
+
+    bool is_constant() const {
+        if( equals_statistic_ && ( *equals_statistic_ )->monotonicity() != monotonically::constant ) {
+            return false;
+        }
+        return true;
```

Comment:
Suggested by clang-tidy readability-simplify-boolean-expr
```suggestion
        return !equals_statistic_ || ( *equals_statistic_ )->monotonicity() == monotonically::constant );
```